roles/bootstrap/tasks/application-credential.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Assert cifmw_bootstrap_cloud_name is defined
+  ansible.builtin.assert:
+    that:
+      - cifmw_bootstrap_cloud_name is defined
+    msg: "Cloud name is mandatory"
+    quiet: true
+
+- name: Create application credential
+  application_credential:
+    name: "cifmw-bootstrap-{{ zuul.build }}"
+    # TODO(sbaker) Ideally expires_at will be set to the job timeout, if that is available here
```

Comment:
I think you have that in `zuul.timeout` ?

Reply:
I figured, I'd still like this to be a TODO :)